ydb/core/io_formats/ydb_dump/csv_ydb_dump.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <ydb/core/scheme/scheme_tablecell.h>
+#include <ydb/core/scheme_types/scheme_type_info.h>
+
+#include <util/generic/strbuf.h>
+#include <util/generic/string.h>
+#include <util/memory/pool.h>
+
+namespace NKikimr::NFormats {
+
+struct TYdbDump {
+    // Parse YdbDump-formatted line
```

Comment:
It is unclear either numBytes will be incremented or overwritten by this function without looking at implementation. At least leave a comment.